static/js/components/SearchBox.jsx

```diff
@@ -1,9 +1,20 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
 
-import * as Actions from '../ducks/sources';
+import {
+  Paper,
+  Button,
+  ButtonGroup,
+  Checkbox,
+  FormControlLabel,
+  TextField,
+} from "@material-ui/core";
```

Comment:
We should use the explicit Material UI imports, as described in their documentation and used elsewhere in SkyPortal, to keep the JS bundle size down.